platform/openide.actions/src/org/openide/actions/HeapView.java

```diff
@@ -47,301 +42,104 @@
-     */
-    private static final int BORDER_H = 4;
-    
-    /**
-     * Colors for the grid. This is alternating pairs for a linear gradient.
-     */
-    private static final Color[] GRID_COLORS = new Color[8];
-    
-    /**
-     * Border color.
-     */
-    private static Color border1Color;
 
     /**
-     * Border color of line below the top.
+     * Foreground color for the chart.
      */
-    private static Color border2Color;
+    private static final Color CHART_COLOR;
 
-    private static Color border3Color;
-    
     /**
-     * Start color for the tick gradient.
+     * Color for text.
      */
-    private static Color minTickColor;
+    private static final Color TEXT_COLOR;
 
     /**
-     * End color for the tick gradient.
+     * Color for the background.
      */
-    private static Color maxTickColor;
+    private static final Color BACKGROUND_COLOR;
 
     /**
-     * Color for the text before blurred.
+     * Number of samples to retain in history.
      */
-    private static Color textBlurColor;
+    private static final int GRAPH_COUNT = 100;
     
     /**
-     * Color for text drawn on top of blurred text.
+     * Key for the Show Text preference.
      */
-    private static Color textColor;
-
-    /**
-     * Start color for the background gradient.
-     */
-    private static Color background1Color;
-
-    /**
-     * End color for the background gradient.
-     */
-    private static Color background2Color;
-    
-    /**
-     * Size used for Kernel used to generate drop shadow.
-     */
-    private static final int KERNEL_SIZE = 3;
-
-    /**
-     * Factor used for Kernel used to generate drop shadow.
-     */
-    private static final float BLUR_FACTOR = 0.1f;
-    
-    /**
-     * How far to shift the drop shadow along the horizontal axis.
-     */
-    private static final int SHIFT_X = 0;
-
-    /**
-     * How far to shift the drop shadow along the vertical axis.
-     */
-    private static final int SHIFT_Y = 1;
+    private static final String SHOW_TEXT = "showText";
 
     static {
         //init colors
-        Color c = UIManager.getColor( "nb.heapview.border1" ); //NOI18N
-        if( null == c )
-            c = new Color(0xA6A295);
-        border1Color = c;
-
-        c = UIManager.getColor( "nb.heapview.border2" ); //NOI18N
-        if( null == c )
-            c = new Color(0xC0BCAD);
-        border2Color = c;
-
-        c = UIManager.getColor( "nb.heapview.border3" ); //NOI18N
-        if( null == c )
-            c = Color.WHITE;
-        border3Color = c;
-
-        c = UIManager.getColor( "nb.heapview.mintick.color" ); //NOI18N
-        if( null == c )
-            c = new Color(0xC7D6AD);
-        minTickColor = c;
-
-        c = UIManager.getColor( "nb.heapview.maxtick.color" ); //NOI18N
-        if( null == c )
+        Color c = UIManager.getColor("nb.heapview.maxtick.color"); //NOI18N
+        if (null == c) {
             c = new Color(0x615d0f);
```

Comment:
For this one, trying using 0x2e90e8 instead. That's the lighter blue color from the NetBeans logo cube. (Might also want to grep the codebase for customizations of nb.heapview.maxtick.color, and change it there as well.)